docs/navigation-configuration.md

```diff
@@ -73,6 +73,7 @@ window.Luigi.setConfig({
 - **context** sends the specified object as context to the view. Use this parameter in combination with the dynamic **pathSegment** to receive the context through the context listeners of **Luigi client**. This is an alternative to using the dynamic value in the **viewUrl**.
 - **defaultChildNode** sets the child node that Luigi activates automatically if the current node has no **viewUrl** defined. Provide **pathSegment** of the child node you want to activate as a string.
 - **isolateView** renders the view in a new frame when you enter and leave the Node. This setting overrides the same-domain frame re-usage. The **isolateView** is disabled by default.
+- **keepSelectedForChildren** fixates the navigation in its current hierarchy, omitting display of children
```

Comment:
``- **keepSelectedForChildren** focuses the navigation in its current hierarchy, omitting the display of children.``
I wouldn't use "fixate" here tbh.